### DISCUSSION TITLE
Rule to detect/ban deprecated stuff?

### DISCUSSION THREAD

**Discussion by jpike88** [biomejs/biome#1499] [Ideas]
<img width="469" alt="Screenshot 2024-01-10 at 10 00 36 AM" src="https://github.com/biomejs/biome/assets/9585787/8ca7930b-ec67-472d-9ee6-433ced3ba8eb">

See above case, where window.event was accidentally called. That is a deprecated property, would be nice to just ban all uses of deprecated properties.

**Comment by Conaclos** [comment]
This is already planned in #3
